turtleflags.py

```diff
@@ -266,14 +266,14 @@ def _parse_command_line(self):
         try:
             opts, args = getopt.getopt(argv[1:], 'hor',
                                        ['help', 'output_png', 'run'])
-        except getopt.GetoptError, err:
-            print str(err)
-            print self._HELP_MSG
+        except getopt.GetoptError as err:
+            print(str(err))
```

Comment:
You can also print `err.msg` instead of converting err to a string.

Reply:
Done in b13917f